app/views/user_mailer/admin_hidden_work_notification.text.erb

```diff
@@ -1,12 +1,12 @@
 <% content_for :message do %>
-    Dear <%= @user.default_pseud.byline %>,
+    <%= t('.hello', user: @user.default_pseud.byline) %>
 
-    Your work "<%= @work.title.html_safe %>" (<%= work_url(@work) %>) has been reported to the Abuse Team as a violation of the Archive of Our Own's Terms of Service (<%= root_url + "tos" %>).
+    <%= t('.text.part1', title: @work.title, tos_url: root_url + "tos" ) %> 
```

Comment:
There's an extra space before the `)` here. Also, can this be `tos_url` instead of `root_url + "tos"`?